<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-I8PQN7U" name="Validating Title" tests="1" failures="1" timestamp="14 May 2020 21:01:16 GMT" time="17.709" errors="0">
  <testcase name="validateTitle" time="1.704" classname="academy3.TitleValidation3">
    <failure type="java.lang.AssertionError" message="expected [2FEATURED COURSES] but found [FEATURED COURSES]">
      <![CDATA[java.lang.AssertionError: expected [2FEATURED COURSES] but found [FEATURED COURSES]
	at academy3.TitleValidation3.validateTitle(TitleValidation3.java:33)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- validateTitle -->
</testsuite> <!-- Validating Title -->
